engineapi/proxy.go

```diff
@@ -81,9 +84,19 @@ func NewEngineClientProxy(im *longhorn.InstanceManager, logger logrus.FieldLogge
 	}
 
 	ctx, cancel := context.WithCancel(context.Background())
-	client, err := imclient.NewProxyClient(ctx, cancel, im.Status.IP, InstanceManagerProxyServiceDefaultPort)
+	client, err := imclient.NewProxyClientWithTLS(ctx, cancel, im.Status.IP, InstanceManagerProxyServiceDefaultPort,
+		filepath.Join(types.TLSDirectoryInContainer, types.TLSCAFile),
+		filepath.Join(types.TLSDirectoryInContainer, types.TLSCertFile),
+		filepath.Join(types.TLSDirectoryInContainer, types.TLSKeyFile),
+		"longhorn-backend.longhorn-system")
```

Comment:
Why `longhorn-backend.longhorn-system`?

Reply:
[Server uses `longhorn-backend.longhorn-system` as the peer name.](https://github.com/longhorn/longhorn-instance-manager/blob/master/app/cmd/start.go#L195)